app/src/main/java/ch/epfl/favo/favor/FavorStatus.java

```diff
@@ -8,7 +8,9 @@
   CANCELLED_ACCEPTER("Cancelled by accepter", 4),
   SUCCESSFULLY_COMPLETED("Succesfully Completed", 5),
   ACCEPTED_BY_OTHER("Accepted by other", 6), // additional state
-  EDIT("Edit mode", 7);
+  EDIT("Edit mode", 7),
+  COMPLETED_REQUESTER("Completed by requester", 8),
```

Comment:
We should move EDIT and ACCEPTED BY OTHER to the bottom since they are not really used in the DB

Reply:
wil fix it later